x-pack/plugins/case/server/routes/api/cases/patch_cases.ts

```diff
@@ -60,18 +73,31 @@ export function initPatchCasesApi({ caseService, router }: RouteDeps) {
         });
         if (updateFilterCases.length > 0) {
           const updatedBy = await caseService.getUser({ request, response });
-          const { full_name, username } = updatedBy;
+          const { email, full_name, username } = updatedBy;
           const updatedDt = new Date().toISOString();
           const updatedCases = await caseService.patchCases({
             client: context.core.savedObjects.client,
             cases: updateFilterCases.map(thisCase => {
               const { id: caseId, version, ...updateCaseAttributes } = thisCase;
+              let closedInfo = {};
+              if (updateCaseAttributes.status && updateCaseAttributes.status === 'closed') {
+                closedInfo = {
+                  closed_at: updatedDt,
+                  closed_by: { email, full_name, username },
+                };
+              } else if (updateCaseAttributes.status && updateCaseAttributes.status === 'open') {
+                closedInfo = {
```

Comment:
Do we need it with empty values?

Reply:
I'd like it to be. Otherwise a case will be open and give 'closed_at' times... isnt that weird? We'll have this all tracked in user_actions anyways I think??